GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
             }
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("Building Instant Game with Quick Deploy", EditorStyles.boldLabel);
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Building an instant game with Quick Deploy will significantly reduce the size" +
```

Comment:
`Quick Deploy can significantly reduce the size of a Unity-based instant app by packaging some assets in an AssetBundle that is retrieved from a server during app startup.`

Reply:
Resolved.

Reply:
Unless I'm misreading something, this wasn't done.

Reply:
Oops, had left out one commit by accident. Resolved now.